src/reactor.js

```diff
@@ -24,11 +26,24 @@ import {
 class Reactor {
   constructor(config = {}) {
     const debug = !!config.debug
+    const useCache = config.useCache === undefined ? true : !!config.useCache
+
+    // use default # of items in cache
+    let maxItemsToCache = DEFAULT_MAX_ITEMS_TO_CACHE
+
+    // if user has defined maxItemsToCache, use the number provide or set to null (ie no max)
+    if (config.maxItemsToCache !== undefined) {
+      maxItemsToCache = Number(config.maxItemsToCache)
+      maxItemsToCache = maxItemsToCache && maxItemsToCache > 0 ? maxItemsToCache : null
```

Comment:
minor: is there a difference between `maxItemsToCache && maxItemsToCache > 0` and just `maxItemsToCache > 0` here? looks like it could be simplified